pandas/core/indexes/datetimes.py

```diff
@@ -1985,23 +1985,20 @@ def tz_localize(self, tz, ambiguous='raise', errors='raise'):
         we perform reverse operation where we remove tize zone & make it
         tz-naive
 
-        In [2]: dti = pd.date_range('2018-03-01', '2018-03-05')
+        >>> dti = pd.date_range('2018-03-01', '2018-03-05')
 
-        In [3]: dti
-
-        Out[3]: DatetimeIndex(['2018-03-01', '2018-03-02', '2018-03-03',
+        >>> dti
+        DatetimeIndex(['2018-03-01', '2018-03-02', '2018-03-03',
         '2018-03-04', '2018-03-05'], dtype='datetime64[ns]', freq='D')
 
-        In [4]: dti.tz_localize(tz='US/Eastern')
-
-        Out[4]: DatetimeIndex(['2018-03-01 00:00:00-05:00',
+        >>> dti.tz_localize(tz='US/Eastern')
+        DatetimeIndex(['2018-03-01 00:00:00-05:00',
         '2018-03-02 00:00:00-05:00', '2018-03-03 00:00:00-05:00',
         '2018-03-04 00:00:00-05:00', '2018-03-05 00:00:00-05:00'],
         dtype='datetime64[ns, US/Eastern]', freq='D')
 
-        In [5]: dti.tz_localize('US/Eastern').tz_localize(None)
-
-        Out[5]: DatetimeIndex(['2018-03-01', '2018-03-02', '2018-03-03',
+        >>> dti.tz_localize('US/Eastern').tz_localize(None)
+        DatetimeIndex(['2018-03-01', '2018-03-02', '2018-03-03',
         '2018-03-04', '2018-03-05'], dtype='datetime64[ns]', freq='D')
```

Comment:
More a personal opinion, but couple of things that I think would make the examples more clear:
- Use 3 dates instead of 5
- Add short sentences between commands on what's going on (e.g. "The values for the times are the same, but the index contains the timezone")
- Save the output of `dti.tz_localize(tz='US/Eastern')` so it can be reused in the last command, instead of doing to two localizations in the same line. May be naming `dti` `tz_naive` first, and `tz_aware` later would make them more clear too.

Reply:
okey sure